docs/building-your-bot-on-iadvize.md

```diff
@@ -752,8 +752,23 @@ In order to provide a great conversational experience here are some of the best
 
 ⚠️ Please note that these guidelines are important for your app to pass the [iAdvize review process](/documentation/getting-started#app-reviewing-process). ⚠️
 
-#### Awaits on Bot Transfers
-As of now our bot transfer process allows you to set any `await`s time you might like. Please be advised that you should not set an await time inferior to 15 seconds. Under that delay our system will not have a sufficient amount of time to verify availability. This delay if perfectly acceptable in term of conversational experience and is required for the transfer to operate correctly.
+#### Awaits / timeout on Bot Transfers
+Our bot transfer process allows you to set any `timeout` value you might like. Please be advised that you should not set a timeout value inferior to 15 seconds. Under that delay our system will not have a sufficient amount of time to verify availability. This delay if perfectly acceptable in term of conversational experience and is required for the transfer to operate correctly.
```

Comment:
"This delay **is** perfectly..."

Tu peux rajouter que si "transferOptions" n'est pas renseigné, la valeur par défaut est de 60 secondes.

Reply:
Bien vu pour la typo.
Okay ça marche